respa_admin/views/prices.py

```diff
@@ -224,3 +229,28 @@ class PriceListDeleteView(ExtraContextMixin, DeleteView):
     def get_queryset(self):
         qs = super().get_queryset()
         return qs.modifiable_by(self.request.user)
+
+
+class PriceListCopyView(PriceListCreateView):
+    def get(self, request, *args, **kwargs):
+        self.object = None
+        id = self.kwargs.get("price_list_id")
```

Comment:
Minor point: should be e.g. `original_price_list_id` or something as `id` is a reserved Python keyword

Reply:
Yeah, good point. I don't even need to use a variable here.

Reply:
Fixed